typescript/test/data/deposit-sweep.ts

```diff
@@ -350,24 +350,23 @@ export const depositSweepWithNonWitnessMainUtxoAndWitnessOutput: DepositSweepTes
     witness: true,
     expectedSweep: {
       transactionHash: TransactionHash.from(
-        "7831d0dfde7e160f3b9bb66c433710f0d3110d73ea78b9db65e81c091a6718a0"
```

Comment:
Why this changed?

Reply:
The change is the result of how we order inputs in the transaction.
When we used `bcoin`, the input order was decided by the `bcoin`'s `fund` function.
After we got rid of `bcoin` we add the inputs manually: main UTXO first, deposit UTXOs second.

Luckily, `bcoin` usually sorted the inputs in the same way, with main UTXO first followed by deposits.
Therefore most unit tests didn't have to be updated.
The unit test  in question is an exception: the deposit UTXO was first, followed by the main UTXO.
The order of inputs influence signatures and transaction hash in the end.

To verify that the problem is related only to the order of inputs, I temporarily changed the order in the same way as `bcoin` would do. The test in the original form passed.
Also, I decoded the raw transaction and compared it with the original raw transaction and the only difference I noticed was the input order.

Reply:
Makes sense! Thanks!